ops/deploy-ccs.groovy

```diff
@@ -53,7 +53,7 @@ class AmiIds implements Serializable {
  * @return a new {@link AmiIds} instance detailing the already-existing, latest AMIs (if any) that are now available for use
  * @throws RuntimeException An exception will be bubbled up if the AMI-builder tooling returns a non-zero exit code.
 */
-def findAmis() {
+def findAmis(String branchName = 'master') {
```

Comment:
Could we extend this function to also find the server regression AMI? And then reference it in the same way as the others in the `Jenkinsfile`?

Reply:
I looked at our list of amis on AWS and didnt find a specific labeled server regression ami. The functionality here should suffice.